examples/otel-operator/instrumentation.yaml

```diff
@@ -8,13 +8,18 @@ spec:
     #endpoint: http://mockotlpserver:4317
     # HTTP/proto (used by default by EDOT)
     endpoint: http://mockotlpserver:4318
+  sampler:
+    type: parentbased_traceidratio
+    argument: "1.0"
```

Comment:
reviewer note: Even thought `sampler` is option in the `Instrumentation` config, the OTel Operator code spits out a warning if `sampler` isn't defined. This is just to avoid the warning.